simple-mbed-cloud-client.cpp

```diff
@@ -147,22 +163,11 @@ int SimpleMbedCloudClient::init() {
     palStatus_t status = PAL_SUCCESS;
     status = pal_fsRmFiles(DEFAULT_FIRMWARE_PATH);
     if(status == PAL_SUCCESS) {
-        printf("[SMCC] Firmware storage erased.\n");
+        tr_info("Firmware storage erased");
     } else if (status == PAL_ERR_FS_NO_PATH) {
-        printf("[SMCC] Firmware path not found/does not exist.\n");
+        tr_info("Firmware path not found/does not exist");
     } else {
-        printf("[SMCC] Firmware storage erasing failed with %" PRId32, status);
-        return 1;
-    }
-#endif
-
-#if MBED_CONF_APP_DEVELOPER_MODE == 1
-    printf("[SMCC] Starting developer flow\n");
-    fcc_status = fcc_developer_flow();
-    if (fcc_status == FCC_STATUS_KCM_FILE_EXIST_ERROR) {
-        printf("[SMCC] Developer credentials already exist\n");
```

Comment:
Could you re-enable this message? It's really useful to us to understand whether credentials are reused, or new ones are installed

Reply:
They're still here, but in `SimpleMbedCloudClient::verify_cloud_configuration` and in a trace call.